tests/beman/execution/CMakeLists.txt

```diff
@@ -105,51 +98,9 @@ if(NOT DEFINED CMAKE_CXX_STANDARD)
   set(CMAKE_CXX_STANDARD 23)
 endif()
 
-if(NOT DEFINED CMAKE_PREFIX_PATH)
-  set(CMAKE_PREFIX_PATH ${CMAKE_INSTALL_PREFIX})
-endif()
-
-if(PROJECT_IS_TOP_LEVEL)
-  find_package(beman_execution 0.0.1 EXACT QUIET)
-  if(beman_execution_FOUND)
-    set(execution_tests exec-awaitable.test) # only one sample to save time! CK
-  else()
-    set(BEMAN_EXECUTION_ENABLE_INSTALL ON)
-    add_subdirectory(../../.. beman_execution)
-
-    include(CMakePrintHelpers)
-    cmake_print_variables(TARGET_ALIAS TARGET_LIBRARY TARGET_PREFIX PROJECT_SOURCE_DIR)
-  endif()
-endif()
-
-remove_definitions(-DNDEBUG) # NOTE: we want ASSERT statements in Release too! CK
-
 foreach(test ${execution_tests})
-  set(TEST_EXE ${TARGET_PREFIX}.${test})
+  set(TEST_EXE beman.execution.${test})
   add_executable(${TEST_EXE} ${test}.cpp)
   target_link_libraries(${TEST_EXE} PRIVATE beman::execution)
   add_test(NAME ${TEST_EXE} COMMAND $<TARGET_FILE:${TEST_EXE}>)
 endforeach()
-
-if(NOT PROJECT_IS_TOP_LEVEL AND BEMAN_EXECUTION_ENABLE_TESTING)
-  # test if the targets are findable from the build directory
-  # cmake-format: off
-  add_test(NAME find-package-test
-    COMMAND ${CMAKE_CTEST_COMMAND}
-      # --verbose
-      --output-on-failure
-      -C $<CONFIG>
-      --build-and-test
-          "${CMAKE_CURRENT_SOURCE_DIR}"
-          "${CMAKE_CURRENT_BINARY_DIR}/find-package-test"
-      --build-generator ${CMAKE_GENERATOR}
-      --build-makeprogram ${CMAKE_MAKE_PROGRAM}
```

Comment:
... den sie wissen nicht was sie tun!

Reply:
Similar to my previous comment, I admit that I don't understand what this test tries to achieve. In my development it was always executed (and I haven't figured out, yet, why) and took a substantial amount of time.

Reply:
with https://github.com/bemanproject/exemplar/pull/173 this variant, only examples are used.